addons/im_livechat/static/src/legacy/public_models/messaging.js

```diff
@@ -1,6 +1,6 @@
 /** @odoo-module **/
 
-import { one, Patch } from "@mail/legacy/model";
+import { one, Patch } from "@im_livechat/legacy/model";
```

Comment:
note for after: patches can be merged into main files as they are moved to this module